lib/radiator_web/views/api/chapters_view.ex

```diff
@@ -1,14 +1,22 @@
 defmodule RadiatorWeb.Api.ChaptersView do
   use RadiatorWeb, :view
 
-  alias HAL.{Document, Link}
+  alias HAL.{Document, Link, Embed}
   alias Radiator.AudioMeta.Chapter
 
+  def render("index.json", assigns) do
+    %Document{}
+    |> Document.add_embed(%Embed{
+      resource: "rad:chapter",
```

Comment:
haven't looked at the result. But wouldn't it be rad:chapters as container? I'm unclear what a top level container should look like here though.

Reply:
I think the resource name is always supposed to be singular. The example in http://stateless.co/hal_specification.html includes `ea:order`, which is a list.